src/plugins/data/server/index_patterns/routes/create_index_pattern.ts

```diff
@@ -15,9 +15,7 @@ import type { DataPluginStart, DataPluginStartDependencies } from '../../plugin'
 
 const indexPatternSpecSchema = schema.object({
   title: schema.string(),
-
   id: schema.maybe(schema.string()),
-  version: schema.maybe(schema.string()),
```

Comment:
I am not sure how the version makes sense here. Different version is set when object get saved using so client.

Reply:
Its useful to have this if you want to determine whether you have the current version of a saved object.

Reply:
reverting 👍